tests/modules/services/redshift-gammastep/redshift-basic-configuration-expected.service

```diff
@@ -2,7 +2,7 @@
 WantedBy=graphical-session.target
 
 [Service]
-ExecStart=@redshift@/bin/redshift -c /home/hm-user/.config/redshift/redshift.conf
+ExecStart=@redshift@
```

Comment:
How to properly match here to prevent the following error:

```patch
redshift-basic-configuration: FAILED
Expected home-files/.config/systemd/user/redshift.service to be same as /nix/store/6psd0mzcxigyv05s199hsp8m17b98zm1-redshift-basic-configuration-expected.service but were different:
--- actual
+++ expected
@@ -2,7 +2,7 @@
 WantedBy=graphical-session.target
 
 [Service]
-ExecStart=/nix/store/f76092w1ll8d121apa09l2n5smwgmjax-redshift
+ExecStart=@redshift@
 Restart=on-failure
 RestartSec=3
 
For further reference please introspect /nix/store/0wz59xpck17n6yp8rpliil0vcmwbhcrh-nmt-report-redshift-basic-configuration
```